packages/beacon-state-transition/src/fast/util/epochProcess.ts

```diff
@@ -8,8 +8,8 @@ import {
   computeStartSlotAtEpoch,
   getChurnLimit,
   isActiveValidator,
-} from "../../../util";
-import {FAR_FUTURE_EPOCH} from "../../../constants";
+} from "../../util";
+import {FAR_FUTURE_EPOCH} from "../../constants";
 import {
```

Comment:
The `statusProcessEpoch` function below seems to be better to put into `phase0` folder, it's fine for now through